frb_codegen/src/entrypoint/dart.rs

```diff
@@ -151,12 +150,20 @@ pub(crate) fn generate_dart_code(
     Ok(())
 }
 
+pub fn generate_dart_common_definitions(configs: &[Opts]) -> anyhow::Result<()> {
+    let common_definitions = generate_common_definitions_code(configs);
+    write_dart_definitions_to_dart_decl(
+        &configs[0].dart_decl_output_path.clone().unwrap(),
```

Comment:
nit: Are we guaranteed that `dart_decl_output_path` exists and only exists in configs[0]? If so we can add an sanity check assertion

Reply:
We guaranteed that dart_decl_output_path exists, but I added the sanity check assertion anyway.